src/lime/_internal/backend/native/NativeAudioSource.hx

```diff
@@ -199,47 +149,35 @@ class NativeAudioSource
 		AL.sourcePause(handle);
 
 		if (streamTimer != null)
-		{
```

Comment:
Please don't remove braces like this. It makes it more difficult to filter through the PR to find the truly relevant parts. Additionally, though, while you may personally prefer to omit braces, others prefer to always include braces, even when optional, so best not to change what may have been an intentional choice.

Reply:
Ok